.travis.yml

```diff
@@ -17,16 +17,12 @@ env:
         - EXTRA_ARGV="'--disable-pytest-warnings'"
 
 language: python
-# Default Python version is usually 2.7
-python: 3.6
-sudo: required
+python: 3.7
 dist: bionic
 services: docker
+os: linux
 
-matrix:
-  exclude:
-      # Exclude the default Python 3.6 build
-      - python: 3.6
+jobs:
```

Comment:
lines 19-here update the confusing travis.yml: we don't have any python3.6 jobs to exclude, `sudo` is no longer needed, `matrix` should be replaced by `jobs`.